sftp/1.3/modules/ROOT/pages/sftp-list.adoc

```diff
@@ -30,88 +49,124 @@ The following example lists the contents of a folder and all messages in `direct
+. Set the *Connector configuration* field to the previously configured connection in the *List* operation.
+. Set the *Path* field to `#[attributes.path]`.
+
+In the XML editor, the configuration looks like this:
 
 [source,xml,linenums]
 ----
-<flow name="poll">
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:ftps="http://www.mulesoft.org/schema/mule/ftps" xmlns:file="http://www.mulesoft.org/schema/mule/file"
+	xmlns:sftp="http://www.mulesoft.org/schema/mule/sftp"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/sftp http://www.mulesoft.org/schema/mule/sftp/current/mule-sftp.xsd
+http://www.mulesoft.org/schema/mule/file http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
+http://www.mulesoft.org/schema/mule/ftps http://www.mulesoft.org/schema/mule/ftps/current/mule-ftps.xsd">
+	<sftp:config name="SFTP_Config" doc:name="SFTP Config" >
+		<sftp:connection host="localhost"/>
+	</sftp:config>
+	<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" >
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<flow name="poll">
   <scheduler>
       <scheduling-strategy>
           <fixed-frequency frequency="1000"/>
       </scheduling-strategy>
   </scheduler>
-  <sftp:list directoryPath="~/dropFolder" />
+  <sftp:list directoryPath="~/dropFolder" config-ref="SFTP_Config">
+			<sftp:matcher />
+		</sftp:list>
   <foreach>
       <flow-ref name="processFile" />
-      <sftp:delete path="#[attributes.path]" />
+      <sftp:delete path="#[attributes.path]" config-ref="SFTP_Config"/>
   </foreach>
 </flow>
+</mule>
 ----
 
-In the previous example, a flow lists the contents of a folder once per second. The flow then processes the files one by one, deleting each file after it is processed because there is a `delete` operation in the `<foreach>` component.
-
-For automatic polling, you can use the *On New or Updated File* component, which also allows you to optionally set post-processing options to automatically move or delete files.
-
 == Match Filter
 
-When listing files, use the `<sftp:matcher>` element so that only files that match the specified `matcher` criteria are accepted. This element defines the possible criteria to use to either accept or reject a file.
+When listing files, use the *File Matching Rules* parameter so that only files that match the specified criteria are accepted. This parameter defines the possible attributes to use to either accept or reject a file.
```

Comment:
Suggestion:
When listing files, use the *File Matching Rules* parameter that accepts files that match the specified criteria. This parameter defines the possible attributes to either accept or reject a file.